lib/countly.js

```diff
@@ -172,7 +173,7 @@ Countly.Bulk = Bulk;
             // Common module debug value is set to init time debug value
             cc.debug = conf.debug;
 
-            CountlyStorage.initStorage(conf.storage_path, conf.storage_type);
+            CountlyStorage.initStorage(conf.storage_path, conf.storage_type, false, false, conf.custom_storage_method);
```

Comment:
you might move custom storage method to the 3rd parameter to avoid giving the default params. However, if it is published like that no problemo

Reply:
in the case of countly.js those params will be false anyway, would look prettier but doesn't do much I guess

Reply:
🆗